modules/nextflow/src/main/groovy/nextflow/script/ExecutionStack.groovy

```diff
@@ -89,3 +131,56 @@ class ExecutionStack {
     }
 
 }
+
+@CompileStatic
+class ERROR_IN_NEXTFLOW_PIPELINE extends Throwable {
```

Comment:
The use of all uppercase class names is very unusual for Groovy/Java code convention. Camel case should be used instead